pychromecast/socket_client.py

```diff
@@ -212,6 +217,21 @@ def __init__(self, host, port=None, cast_type=CAST_TYPE_CHROMECAST,
                                  NetworkAddress(self.host, self.port)))
             raise
 
+    def _get_host_from_service(self, service):
+        """ Resolve host and port from service. """
+        host = self.host
+        port = self.port
+        if service:
```

Comment:
if service is `None`, we're connecting directly to a host name or IP-address

Reply:
Can you add this as a comment in the code?